src/main/java/mealplan/MealplanBoxUI.java

```diff
@@ -78,11 +74,8 @@ public void actionPerformed( ActionEvent e ) {
         JButton button_cal = new JButton( new AbstractAction("calculate calories") {
             @Override
             public void actionPerformed( ActionEvent e ) {
-                try {
-                    appController.getMealplanController().getCalories();
-                } catch (IOException ex) {
-                    throw new RuntimeException(ex);
-                }
+                String username = appController.getLoginControllor().preformGetLoggedInUser();
```

Comment:
Should this be renamed to performGetLoggedInUser instead of preformGetLoggedInUser?